tests/api_fastapi/core_api/routes/public/test_backfills.py

```diff
@@ -267,6 +267,53 @@ def test_no_schedule_dag(self, session, dag_maker, test_client):
         assert response.status_code == 409
         assert response.json().get("detail") == f"{dag.dag_id} has no schedule"
 
+    @pytest.mark.parametrize(
+        "repro_act, repro_exp,run_backwards, status_code",
+        [
+            ("none", ReprocessBehavior.NONE, False, 409),
+            ("completed", ReprocessBehavior.COMPLETED, False, 200),
+            ("completed", ReprocessBehavior.COMPLETED, True, 409),
+        ],
+    )
+    def test_create_backfill_with_depends_on_past(
+        self, repro_act, repro_exp, run_backwards, status_code, session, dag_maker, test_client
+    ):
+        with dag_maker(session=session, dag_id="TEST_DAG_1", schedule="0 * * * *") as dag:
+            EmptyOperator(task_id="mytask", depends_on_past=True)
+        session.query(DagModel).all()
+        session.commit()
```

Comment:
What are those two lines for ? Query without using any results, and committing an empty session I believe.